### DISCUSSION TITLE
How to show mesh boundaries in domain?

### DISCUSSION THREAD

**Discussion by bjmvandongen** [firemodels/fds#13770] [Q&A]
I cannot find in the user guide of smokesview how to show the meshes in the domain. 
I think I have tried everything.

Does anyone know how i can show the meshes in smokesview?

**Comment by gforney** [comment]
the 'o' key toggles between various outline modes.  one mode is to show
mesh outlines.  the 'm' key will toggle the mesh that is highlighted.
select the Show/Hide>Labels>Mesh menu item to output the mesh index that is
highlighted.  Appendix C documents keyboard shortcuts.

On Mon, Nov 18, 2024 at 3:31 AM bjmvandongen ***@***.***>
wrote:

> I cannot find in the user guide of smokesview how to show the meshes in
> the domain.
> I think I have tried everything.
>
> Does anyone know how i can show the meshes in smokesview?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/13770>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC6UCRWIGVZD54X7TJ4DWKT2BGQVPAVCNFSM6AAAAABR7CEAOOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGUYDKMBSGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Glenn Forney

**Reply by bjmvandongen** [reply]
Thank you very much! I can now see the meshes!

**Comment by gforney** [comment]
good.  let me know if you have any other questions

On Mon, Nov 18, 2024 at 2:58 PM bjmvandongen ***@***.***>
wrote:

> Thank you very much! I can now see the meshes!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/13770#discussioncomment-11296353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC6UCRR4LWUDEXSGWPYHVGT2BJBIFAVCNFSM6AAAAABR7CEAOOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMRZGYZTKMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Glenn Forney
